network_node.go

```diff
@@ -21,11 +22,12 @@ import (
 )
 
 type NetworkNode struct {
-	Identity peer.ID // the local node's identity
-	Kad      *kad.IpfsDHT
-	PeerHost host.Host // the network host (server+client)
-	Network  *BasicVideoNetwork
-	streams  map[peer.ID]*BasicStream
+	Identity       peer.ID // the local node's identity
+	Kad            *kad.IpfsDHT
+	PeerHost       host.Host // the network host (server+client)
+	Network        *BasicVideoNetwork
+	outStreams     map[peer.ID]*BasicStream
```

Comment:
Might be a good idea to have a basicInStream and basicOutStream to make things more clear.

Reply:
Added https://github.com/livepeer/go-livepeer/issues/199